deps.ts

```diff
@@ -1,12 +1,12 @@
-export * as Bson from "./bson/bson.ts";
-export { crypto } from "https://deno.land/std@0.117.0/crypto/mod.ts";
-export { BufReader } from "https://deno.land/std@0.117.0/io/mod.ts";
-export { writeAll } from "https://deno.land/std@0.117.0/streams/conversion.ts";
-export { deferred } from "https://deno.land/std@0.117.0/async/deferred.ts";
-export type { Deferred } from "https://deno.land/std@0.117.0/async/deferred.ts";
-export * as b64 from "https://deno.land/std@0.117.0/encoding/base64.ts";
-export * as hex from "https://deno.land/std@0.117.0/encoding/hex.ts";
+export * as Bson from "https://deno.land/x/web_bson@v0.1.2/mod.ts";
```

Comment:
```suggestion
export * from "https://deno.land/x/web_bson@v0.1.2/mod.ts";
```
the MongoDB driver also doesn't wrap it. We should also not wrap it.

Reply:
We can't decide here. i think warping Bson types makes the driver more clean. so better to move this discussion into a separated issue.